pkg/hintrunner/zero/zerohint_uint512.go

```diff
@@ -0,0 +1,101 @@
+package zero
+
+import (
+	"math/big"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+)
+
+const (
+	P_LOW  = "201385395114098847380338600778089168199"
+	P_HIGH = "64323764613183177041862057485226039389"
+)
+
+func createInvModPUint512Hinter(resolver hintReferenceResolver) (hinter.Hinter, error) {
```

Comment:
can you place the `new..` function before the  `create...` one , and add comments to the `new..` function as done for other hints?